prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml

```diff
@@ -85,14 +86,56 @@ postsubmits:
         - docker/prowbazel
         - image
         - push-safe
+
+  - name: post-test-infra-push-authentikos
+    cluster: test-infra-trusted
+    run_if_changed: '^authentikos/Makefile$'
+    decorate: true
+    max_concurrency: 1
+    branches:
+    - master
+    labels:
+      preset-prow-pusher-service-account: "true"
+      preset-enable-dind: "true"
+    annotations:
+      testgrid-create-test-group: "false"
+    spec:
+      containers:
+      - image: gcr.io/istio-testing/build-tools:2019-10-18T22-40-12
+        command:
+        - entrypoint
+        args:
+        - make
+        - -C
+        - authentikos
+        - deploy
+        securityContext:
+          privileged: true
```

Comment:
Thoughts on using GCB? I don't like seeing privileged containers in the trusted cluster.

Reply:
I am not familiar with GCB. How can we leverage it to avoid using `privileged` containers?

Reply:
GCB jobs run in their cluster, not ours. So our cluster remains secure and figuring out how to deal with this problem becomes GCB's problem. Ask Katharine how we're doing this for k8s

Reply:
I moved these jobs over to the generator for now. I will investigate GCB as an alternative once I become more familiar.